lib/charms/opensearch/v0/opensearch_plugin_manager.py

```diff
@@ -54,11 +53,6 @@
         "config": "plugin_opensearch_knn",
         "relation": None,
     },
-    "repository-s3": {
-        "class": OpenSearchBackupPlugin,
-        "config": None,
-        "relation": "s3-credentials",
-    },
```

Comment:
This plugin depends on `keystore`, therefore we cannot set it via plugin_manager. It will be managed by `opensearch-backups`.